### PR TITLE
Finish up production deployment

### PR DESCRIPTION
**Pull request by rogeruiz**
These should be the last remaining bits before unpausing the pipeline to deploy the `uaa-guard-service`.

cc: @rememberlenny

### CODE REVIEW

**Comment by rogeruiz** [comment]
The pipeline has been flown too with `fly sp`. So everything should be good to go once this is merged in we can unease the pipeline.

**Comment by rogeruiz** [comment]
✅ 

- [x] Verify that broker password is what is currently deployed in production for the `credentials.yml`
- [x] ~~Verify that proxy app name matches what is currently deployed in production~~ _REALLY VERIFY THIS_
- [x] Verify that proxy route matches what is currently deployed in production
- [x] Verify that app names match what is currently deployed in production
- [x] Rename broker service is named after broker app deployment because of `cg-pipeline-tasks/register-service-broker.sh`

**Review by jmcarp — APPROVED** [review]

**Pull request merged**
